src/ConnectedMode/Persistence/ISolutionBindingRepository.cs

```diff
@@ -70,5 +86,45 @@ public BoundSonarQubeProject Read(string configFilePath)
 
             return bound;
         }
+
+        public bool Write(string configFilePath, BoundSonarQubeProject binding)
+        {
+            if (binding == null)
```

Comment:
`is null` and `is not null` are semantically better way of checking for nullability.
```suggestion
            if (binding is null)
```

It reads more fluently. And it never calls `.Equals` for records, improving the performance a tiny bit.